ivy/functional/backends/paddle/elementwise.py

```diff
@@ -1365,24 +1184,7 @@ def gcd(
     return paddle.gcd(x1, x2)
 
 
-@with_unsupported_device_and_dtypes(
-    {
-        "2.5.1 and below": {
-            "cpu": (
-                "int8",
-                "int16",
-                "int32",
-                "int64",
-                "uint8",
-                "float16",
-                "float32",
-                "float64",
-                "bool",
-            )
-        }
-    },
-    backend_version,
-)
+@with_supported_dtypes({"2.5.1 and below": "complex"}, backend_version)
```

Comment:
shouldn't this be a `tuple`? Could you please also update similar occurrences in the changes made?

Reply:
I think python consider tuple with 1 element to be the same as 1 element if that makes sense ?

Reply:
Thanks for fixing this